server/index.js

```diff
@@ -66,14 +104,16 @@ app.post('/api/inventory/1', uploadsMiddleware, (req, res, next) => {
     .catch(err => next(err));
 });
 
-app.delete('/api/inventory/1/:articleId', (req, res, next) => {
+app.delete('/api/inventory/:userId/:articleId', (req, res, next) => {
   const articleId = req.params.articleId;
+  const userId = req.params.userId;
```

Comment:
All of these spots where you are using the `userId` from the path parameters need to be updated to use the `req.user.userId` instead. At least, the spots where the user is creating/updating/deleting. otherwise, it's possible for one user to mess with another user's stuff.